types/stripe/index.d.ts

```diff
@@ -767,25 +773,33 @@ declare namespace Stripe {
             /**
              * ID of the customer this charge is for if one exists. [Expandable]
              */
-            customer: string | customers.ICustomer;
+            customer: string | customers.ICustomer | null;
 
             description?: string;
 
+            /**
+             * The account (if any) the charge was made on behalf of, with an automatic
+             * transfer. See the [Connect documentation]
+             * <https://stripe.com/docs/connect/destination-charges> for details.
+             * [Expandable]
+             */
+            destination?: string | null;
```

Comment:
This property is `Expandable` which means it can be expanded into an object (see `source_transfer` on L894 as an example). Can you also add the object interface to the type (creating the interface if need be)?